### ISSUE TITLE
[bug]: Not able to read the full text in dropdown field

### ISSUE THREAD

**Issue by INNOVATIVEGAMER** [strapi/design-system#843]
### What version of strapi-design-system are you using?

**Versions**
- "@strapi/design-system": "^1.2.0",
- "@strapi/strapi": "4.2.0",
- OS : MacOs

### What's Wrong?

We have some long relation inputs in relation dropdown as shown in screenshot below.
The value uses ellipsis if the input text is too large. We are not able to view the input text.

<img width="323" alt="Screenshot 2023-01-17 at 12 12 26 PM" src="https://user-images.githubusercontent.com/55017197/212832912-4ce91bfe-7342-4697-9a06-3fa9b7b35b4a.png">

### To Reproduce

1. Create a Combobox field in design system and add a long text option such that it exceeds its container width.
2. You are not able to read the full text when hovered or selected in Combobox.

Also you can create relations with long text as their relation identifier, and try to select relation in admin panel.

### Expected Behaviour

I think a tooltip showing full value when hovered will be a good solution.

**Comment by INNOVATIVEGAMER** [comment]
Ok, I found a work-through for the problem.
I just changed the width of the relational input combobox to 100% width from the “configure the view” setting.
So the results look like -
Still I find this as workaround and tooltip should be an option if it is getting ellipsis.

Setting - 
<img width="1431" alt="Screenshot 2023-01-17 at 8 27 06 PM" src="https://user-images.githubusercontent.com/55017197/212933388-9f757895-119d-4ca6-bcdd-b602e2b5bb51.png">

Result -
<img width="714" alt="Screenshot 2023-01-17 at 8 22 28 PM" src="https://user-images.githubusercontent.com/55017197/212933494-fb27dfe8-c3e8-4f60-931c-a730c450b4db.png">
